client.go

```diff
@@ -82,7 +82,7 @@ func NewClient(opts ...ClientOption) (*Client, error) {
 			c.mu.Unlock()
 			if ok {
 				if stream.state == running {
-					stream.state = underflow
+					stream.underflow = true
```

Comment:
As the definition of `Underflow()` is `true if any underflows happend since the last call to Start or Resume`, underflow can't be one of the states. (The state should be resumed to `running` when receiving `proto.Started` after the underflow condition is resolved.)
https://github.com/jfreymuth/pulse/blob/981ec0d95a86d75eae47cc5101b53dd1f395b256/playback.go#L174-L177